<?xml version="1.0"?>
<launch>
    <arg name="threhold_of_distance_btw_pc_and_person" default="0.35"/>
    <arg name="range" default="10.0"/>
    <arg name="grid_num" default="2500"/>
    <arg name="grid_width" default="50"/>
	<arg name="people_num" default="2"/>
    <arg name="flow_image_size" default="30"/>
    <arg name="hz" default="10.0"/>
	<arg name="manual_crop_size" default="5"/>
	<arg name="is_annotate_occupancy" default="true"/>
	<arg name="cylinder_radius" default="0.212"/>
	<arg name="cylinder_radius_error" default="0.050"/>

	<param name="use_sim_time" value="false"/>

	<node pkg="bev_evaluator" type="flow_multiarray_publisher" name="flow_multiarray_publisher" output="screen">
		<param name="RANGE" value="$(arg range)" type="double"/>
		<param name="CYLINDER_RADIUS" value="$(arg cylinder_radius)" type="double"/>
		<param name="CYLINDER_RADIUS_ERROR" value="$(arg cylinder_radius_error)" type="double"/>
        <param name="GRID_WIDTH" value="$(arg grid_width)" type="int"/>
		<param name="PEOPLE_NUM" value="$(arg people_num)" type="int"/>
		<param name="Hz" value="$(arg hz)" type="double"/>
		<param name="MANUAL_CROP_SIZE" value="$(arg manual_crop_size)" type="int"/>
		<param name="IS_ANNOTATE_OCCUPANCY" value="$(arg is_annotate_occupancy)" type="bool"/>
        <!-- <rosparam command="load" file="$(find cats)/config/cats_motion_param.yaml"/> -->
	</node>

	<!-- optional -->
	<!-- <include file="$(find bev_evaluator)/launch/bag_player.launch"/> -->
	<include file="$(find velodyne_height_map)/launch/amsl_velodyne_heightmap.launch"/>
	<include file="$(find velodyne_pointcloud)/launch/32e_points.launch"/>

</launch>
